v2/tests/database_collection_operations_test.go

```diff
@@ -98,9 +97,10 @@ func Test_CollectionSetProperties(t *testing.T) {
 	createOpts := arangodb.CreateCollectionProperties{
 		WaitForSync:       false,
 		ReplicationFactor: 2,
-		JournalSize:       1048576 * 2,
-		NumberOfShards:    2,
-		CacheEnabled:      newBool(false),
+		// TODO REMOVE ME - Found unexpected attribute 'journalSize'
```

Comment:
Should it be done in this ticket? If not, then it is better to add a new ticket to backlog